models/Body.js

```diff
@@ -17,8 +17,7 @@ const Body = sequelize.define('body', {
     },
     task_description: {
         type: Sequelize.TEXT,
-        allowNull: true,
-        defaultValue: ''
+        allowNull: true
```

Comment:
In this example, what is the difference between task_description being the empty string and task_description being NULL?

Compare to the `address` field, which can be NULL, but cannot be the empty string.  For the record, the address of [AEGEE-Samara](https://my.aegee.eu/bodies/143) is “we don't have that position”.